.pre-commit-config.yaml

```diff
@@ -47,3 +47,10 @@ repos:
       args:
         - --config
         - pyproject.toml
+
+- repo: https://github.com/asottile/pyupgrade
+  rev: v3.15.0
+  hooks:
+  - id: pyupgrade
+    args: ["--py39-plus"]
```

Comment:
Is this not automatically picked up from `python-requires` in `setup.cfg`?

Reply:
At least I didn't find that documented.

Reply:
> but we could anyways move all of the information to a single file, right?

Yes, it would be good switch to fully switch to `pyproject.toml`, but that's something for another day.